parachain-template/node/Cargo.toml

```diff
@@ -92,7 +92,8 @@ cumulus-client-service = { path = "../../client/service" }
 cumulus-primitives-core = { path = "../../primitives/core" }
 cumulus-primitives-parachain-inherent = { path = "../../primitives/parachain-inherent" }
 cumulus-relay-chain-interface = { path = "../../client/relay-chain-interface" }
-cumulus-relay-chain-local = { path = "../../client/relay-chain-local" }
+cumulus-relay-chain-inprocess-interface = { path = "../../client/relay-chain-inprocess-interface" }
+cumulus-relay-chain-rpc-interface = { path = "../../client/relay-chain-rpc-interface" }
```

Comment:
As this is not stable yet, we should not add this to the template for now.

Reply:
I can remove it here, but are you sure its necessary? It is clearly marked experimental here https://github.com/paritytech/cumulus/pull/963/files#diff-782fb6433e0fb2f09182fc6daeb68222bb0b6c9bff76cacc6ffa23847e26cf70R148 . On removal, we would need to provide a second set of cli arguments (without url argument). And people can only run a full-node with this anyway.

Reply:
FYI: We had a live discussion about this, decided to leave it in.